Justfile

```diff
@@ -0,0 +1,77 @@
+alias b:= build
+alias r:= release
+alias t:= test
+alias f:= fmt
+
+default:
+    just -l
+
+#Building async-minreq (debug)
+[group('dev')]
+build:
+    cargo build
+
+#Building release
+[group('dev')]
+release:
+    cargo build -r
+
+#Formatting source code
+[group('dev')]
+fmt:
+    cargo fmt
+
+#Checking code style and lints
+[group('dev')]
+lint: fmt
+    cargo fmt -- --check

```

Comment:
you don't need the `cargo fmt -- --check` here since your above line already includes the `: fmt` recipe.